From 3a3ee32126c72664e075d0d74c976e5d4c7dda72 Mon Sep 17 00:00:00 2001
From: Nhan Nguyen <nhan.nguyen.yb@renesas.com>
Date: Wed, 14 Nov 2018 16:24:06 +0700
Subject: [PATCH 204/433] media: rcar-dvdec: Change rect value

This change is referred from RCar E2X.
Width and height values are even and compatible with NV16

Signed-off-by: Nhan Nguyen <nhan.nguyen.yb@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/media/platform/rcar-dvdec.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/media/platform/rcar-dvdec.c b/drivers/media/platform/rcar-dvdec.c
index 76834d7..dfc6dc8 100644
--- a/drivers/media/platform/rcar-dvdec.c
+++ b/drivers/media/platform/rcar-dvdec.c
@@ -85,14 +85,14 @@ const struct reg_color rcar_dvdec_reg_color[8] = {
 };
 
 const struct dvdec_rect rcar_dvdec_rect[8] = {
-	{ 16, 241, 0, 1428},	/* 0 : DVDEC_NTSC358 */
-	{ 16, 241, 0, 1428},	/* 1 : DVDEC_NTSC443 */
-	{ 19, 288, 0, 1412},	/* 2 : DVDEC_PAL443 */
-	{ 16, 241, 0, 1428},	/* 3 : DVDEC_PALM */
-	{ 19, 288, 0, 1412},	/* 4 : DVDEC_PALN */
-	{ 19, 288, 0, 1412},	/* 5 : DVDEC_SECAM */
-	{ 16, 241, 0, 1428},	/* 6 : NTSC-443 60 */
-	{ 16, 241, 0, 1428},	/* 7 : PAL 60 */
+	{ 34, 480, 0, 1428},    /* 0 : DVDEC_NTSC358 */
+	{ 34, 480, 0, 1428},    /* 1 : DVDEC_NTSC443 */
+	{ 38, 576, 0, 1412},    /* 2 : DVDEC_PAL443 */
+	{ 34, 480, 0, 1428},    /* 3 : DVDEC_PAL_M */
+	{ 38, 576, 0, 1412},    /* 4 : DVDEC_PAL_N */
+	{ 38, 576, 0, 1412},    /* 5 : DVDEC_SECAM */
+	{ 34, 480, 0, 1428},    /* 6 : NTSC-443 60 */
+	{ 34, 480, 0, 1428},    /* 7 : PAL 60 */
 };
 
 struct rcar_dvdec_color_format {
-- 
2.7.4

